### PR TITLE
v1 support window

### PR DESCRIPTION
**Pull request by krancour**
@technosophos @vdice @radu-matei @karenhchu @bridgetkromhout @squillace

Feed back is welcome not only on the content here, but also the general policy.

### CODE REVIEW

**Review by vdice — APPROVED** [review]

**Review by radu-matei — APPROVED** [review]
LGTM

File: content/2021-11-17-brigade-v1-support-window.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Hello again, Brigadiers!
+
+It's been an exciting week for Brigade and it's only Wednesday. Last Friday we cut the final v2 beta release and announced it here on Monday -- and we're _still_ on track to cut our first release candidate by the end of this week.
+
+With the Brigade 2 GA release just around the corner, it's time to talk about v1 support.
+
+Brigade 2 has been our team's primary focus for about a year and a half now, with the only development work in the v1 line being just a handful of small bug fixes, modest features, and minor Helm chart improvements that were contributed by or requested by the community. These small improvements are as-yet unreleased.
+
+Prior to the Brigade 2 GA release, the maintainers will cut what _may_ be the final releases of both software and Helm charts in the v1 line so that these small improvements finally see the light of day. After that, no further feature development will occur in the v1 lines, but maintainers are committed to providing critical bug fixes and security patches for a window of _six months_, beginning on the date of the Brigade 2 GA release.
```
Comment (bridgetkromhout) [inline content/2021-11-17-brigade-v1-support-window.md:18]: > v1 lines

v1 line? I'm thinking it's singular.

**Comment by bridgetkromhout** [comment]
One clarification, but other than that, this is reasonable. Providing patches for six months gives v1 users time to migrate to v2.

**Comment by krancour** [comment]
> One clarification, but other than that, this is reasonable. Providing patches for six months gives v1 users time to migrate to v2.

@bridgetkromhout excellent point. I will incorporate that.

**Pull request merged**